packages/rest/test/integration/rest.server.integration.ts

```diff
@@ -52,6 +53,28 @@ describe('RestServer (integration)', () => {
     expect(server.url).to.be.undefined();
   });
 
+  it('parses query without decorated rest query params', async () => {
+    // This handler responds with the query object (which is expected to
+    // be parsed by Express)
+    function requestWithQueryHandler(handler: {
```

Comment:
The first argument is context, not handler, and we have a type defined for that: `RequestContext`. 

https://github.com/strongloop/loopback-next/blob/80ba08145df8b7620ea2639af2b6ff67c513d026/packages/rest/src/rest.server.ts#L707

https://github.com/strongloop/loopback-next/blob/80ba08145df8b7620ea2639af2b6ff67c513d026/packages/rest/src/sequence.ts#L14-L21

I am proposing to simplify this part as follows

```ts
function requestWithQueryHandler({request, response}: RequestContext) {
  response.json(request.query);
  response.end();
}
```

Reply:
Fixed